lib/ocpp/v201/charge_point.cpp

```diff
@@ -3603,6 +3616,11 @@ void ChargePoint::cache_cleanup_handler() {
                 EVLOG_debug << "Time based authorization cache cleanup";
             }
             this->auth_cache_cleanup_required = false;
+            lk.unlock();
```

Comment:
There is no need to unlock the lock here since it is running out of scope. That is why there is a "sub" scope here.